### ISSUE TITLE
DNA Scrambler Cannot Revert, and Changes name on PDA, ID and Crew Manifest

### ISSUE THREAD

**Issue by Catofquestionableethics** [space-wizards/space-station-14#32682]
## Description
Implanting and activating the DNA Scrambler implant does not give you an option to revert the implant. Further, it changes your name on the Crew Manifest/Your PDA, as well as your ID.

**Reproduction**
Implant DNA Scrambler. Activate.

**Screenshots**
https://youtu.be/hSxNccTCGXI.

**Additional context**
Discovered and Tested on latest server build, 07/10/2024

**Comment by SlimmSlamm** [comment]
Isn't this intended? If you used a DNA scrambler and were not on the manifest it would be extremely sus and pretty much out you as the former person.

**Comment by thebadman4662** [comment]
> Isn't this intended? If you used a DNA scrambler and were not on the manifest it would be extremely sus and pretty much out you as the former person.

Yes, it made you non-crew unless you could talk your way in into manifest records purge... That is till sec decided to see who was the owner of PDA and ID you have... Since when PDA and ID change though???
